tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -176,6 +176,7 @@
 
 #elif defined(CONFIG_BOARD_NUCLEO_L476RG) || \
 	defined(CONFIG_BOARD_BLACKPILL_F411CE) || \
+	defined(CONFIG_BOARD_BLACKPILL_F401CE) || \
 	defined(CONFIG_BOARD_NUCLEO_L4R5ZI)
```

Comment:
Please make a dedicated commit for this addition please :)

Reply:
Ok, will do.